drivers/bluetooth/hci/ipm_stm32wb.c

```diff
@@ -63,14 +63,14 @@ struct aci_set_ble_addr {
 	uint8_t value[6];
 } __packed;
 
+#ifndef CONFIG_BT_HCI_RAW
```

Comment:
Would it make sense to use `#ifdef CONFIG_BT_HCI_HOST` instead of `#ifndef CONFIG_BT_HCI_RAW`? In practice those should give the same result.

Alternatively, any chance you could use `#if !defined(CONFIG...)` instead of `#ifndef`? Maybe it's just me me, but I read `#ifndef` too easily as `#ifdef` (since that's the common paradigm).

Reply:
Your both points make sense. I'll opt for `#ifdef CONFIG_BT_HCI_HOST` which is more direct from logical perspective.

Reply:
Fixed.